parts/k8s/kubernetesinstalls.sh

```diff
@@ -5,6 +5,10 @@ CC_SOCKET_IN_TMP=/opt/azure/containers/cc-proxy.socket.in
 CNI_CONFIG_DIR="/etc/cni/net.d"
 CNI_BIN_DIR="/opt/cni/bin"
 CNI_DOWNLOADS_DIR="/opt/cni/downloads"
+CONTAINERD_DOWNLOADS_DIR="/opt/containerd/downloads"
+CRI_CONTAINERD_VERSION="1.1.5"
+CONTAINERD_DOWNLOAD_URL="${CONTAINERD_DOWNLOAD_URL_BASE}cri-containerd-${CRI_CONTAINERD_VERSION}.linux-amd64.tar.gz"
+CONTAINERD_TGZ_TMP=$(echo ${CONTAINERD_DOWNLOAD_URL} | cut -d "/" -f 5)
```

Comment:
All of this re-organization is to accommodate `CONTAINERD_TGZ_TMP` re-use across the download and install functions.